src/org/labkey/cds/CDSExportQueryView.java

```diff
@@ -0,0 +1,561 @@
+                            currentRow++;
+                        }
+
+                        rowObject = getRow(sheet, currentRow++);
+                        cell = rowObject.getCell(2, Row.CREATE_NULL_AS_BLANK);
+                        cell.setCellValue(currentFilter);
+                    }
+                    return currentRow + 1;
+                }
+
+                protected Row getRow(Sheet sheet, int rowNumber)
+                {
+                    Row row = sheet.getRow(rowNumber);
+                    if (row == null)
+                    {
+                        row = sheet.createRow(rowNumber);
+                    }
+                    return row;
+                }
+
+                @Override
+                public void renderColumnCaptions(Sheet sheet, List<ExcelColumn> visibleColumns) throws MaxRowsExceededException
+                {
+                    if (!sheet.getSheetName().equals(METADATA_SHEET))
+                        super.renderColumnCaptions(sheet, visibleColumns);
+                }
+            };
+            ew.setFilenamePrefix(getSettings().getQueryName());
+            ew.setAutoSize(true);
+            return ew;
+        }
+        catch (SQLException e)
+        {
+            throw new RuntimeSQLException(e);
+        }
+    }
+
+    private Results createResults(List<List<String>> rowTexts, List<ColumnInfo> columnInfos)
+    {
+        List<Map<String, Object>> rows = new ArrayList<>();
+        if (rowTexts != null)
+        {
+            for (List<String> rowText : rowTexts)
+            {
+                Map<String, Object> row = new HashMap<>();
+                for (int i = 0 ; i < columnInfos.size(); i++)
+                {
+                    row.put(columnInfos.get(i).getAlias(), rowText.get(i));
+                }
+                rows.add(row);
+            }
+        }
+
+        List<LabKeyResultSet.Column> cols = new ArrayList<>();
+        for (ColumnInfo info : columnInfos)
+            cols.add(new LabKeyResultSet.Column(info.getAlias(), String.class));
+
+        ResultSet resultSet = new LabKeyResultSet(rows, cols, null);
+        return new ResultsImpl(resultSet, columnInfos);
+    }
+
+    private List<ColumnInfo> getColumns(List<String> columnNames)
+    {
+
+        List<ColumnInfo> columnInfos = new ArrayList<>();
+        for (String column: columnNames)
+            columnInfos.add(new ColumnInfo(FieldKey.fromParts(column)));
+        return columnInfos;
+    }
+
+    public List<Map<String, Object>> getStudies(String[] studyNames)
+    {
+        List<Map<String, Object>> studies = new ArrayList<>();
+        SimpleFilter filter = new SimpleFilter();
+        filter.addCondition(FieldKey.fromParts(LABEL), Arrays.asList(studyNames), CompareType.IN);
+        SchemaTableInfo table = CDSSchema.getInstance().getSchema().getTable("study");
+
+        try (Results results = new TableSelector(table, getDBColumns(table, STUDY_DB_COLUMNS), filter, null).getResults())
```

Comment:
I think this could be:
new TableSelector(table, getDBColumns(table, STUDY_DB_COLUMNS), filter, null).forEach(studies::add, Map.class);